### ISSUE TITLE
Parse and compare upstream URLs

### ISSUE THREAD

**Issue by AMDmi3** [repology/repology#64]
We need to parse upstream source URLs from repository data. We can perform comparisons based on that too, at least version equality.

**Comment by kolya-ay** [comment]
FYI, [this guys](https://allmychanges.com/) do version parsing along the way. It works with similar stack (python, django). You could probably collaborate (link to changelog, etc).
Thank you for the really usefull project!

**Comment by AMDmi3** [comment]
This idea was implemented in the form of `Related` tab in metapackage view: there for a given metapackage we show other metapackages with intersecting set of upstream URLs.

Regarding AllMyChanges, I've added it to #1 to keep the link around, but it does not seem to be useful.
- It's focused on iOS apps
- Where it works with *nix apps, it seems to be quite outdated (cppcheck 1.43 vs. 1.80, qtox 1.4.0 vs. 1.11.0 etc.)
- There were no commits to their repo since Jan 2016, which suggests that the project is abandoned
